core/src/gl/typedMesh.h

```diff
@@ -37,37 +38,45 @@ class TypedMesh : public VboMesh {
      * after _byteOffset in the mesh vertex data memory
      */
     template<class A>
-    void updateAttribute(GLintptr _byteOffset, unsigned int _nVerts, const A& _newAttributeValue) {
+    void updateAttribute(std::array<int,2> _vertexRange, const A& _newAttributeValue, size_t _attribOffset = 0) {
```

Comment:
Actually the std::array api turned out to be quite handy while trying to change the  to c-arrays. In particular aggregate initialization and copy construction. 

I've now changed it to use a minimal Range type.